pages/setting.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react'
+import Navbar from '../views/Navbar';
+import Footer from '../components/Footer';
+import { BsToggleOn } from 'react-icons/bs';
+
+const Settings = () => {
+
+	const chromeExtensionLink = "https://chrome.google.com/webstore/detail/vibinex/jafgelpkkkopeaefadkdjcmnicgpcncc";
+
+	const list = [
+		{
+			name: 'Enable coverage comments',
+			discription: `If enabled, you'll get coverage comments on each PR`,
+			type: 'toggle',
+			flag: false,
+
+		},
+		{
+			name: 'Enable auto assignment',
+			discription: 'If enabled, it automatically sets the reviewers for each PR',
+			type: 'toggle',
+			flag: false
+
+		},
+
+	];
+
+	const [settingsList, setSettingsList] = useState(list);
+
+	const toggleFlag = (name: string) => {
+		const updatedSettings = settingsList.map((setting) => {
+			if (setting.name === name) {
+				return { ...setting, flag: !setting.flag };
+			}
+			return setting;
+		});
+		setSettingsList(updatedSettings);
+	};
+
+
+	return (
+		<div>
+			<div className='mb-16'>
+				<Navbar ctaLink={chromeExtensionLink} transparent={true} />
+			</div>
+			<div id='pricing' className='w-full py-12 bg-primary-light'>
+				<h2 className='font-bold text-center text-[2rem] mb-4'>Settings</h2>
+
+				<div className='sm:w-[70%] w-[90%] m-auto sm:p-8 p-4 rounded-lg border-2'>
```

Comment:
Why do you keep using these custom '70%' and '90%' widths? The reason why Tailwind has presets is to maintain consistency across the application.